crates/json/src/schema/build.rs

```diff
@@ -613,3 +616,58 @@ impl AnnotationBuilder for CoreAnnotation {
         })
     }
 }
+
+#[cfg(test)]
+mod test {
+    use std::iter::FromIterator;
+
+    use super::{super::build::build_schema, super::CoreAnnotation};
+    use crate::schema::{Application::Properties, Keyword};
+    use serde_json::Map;
+
+    #[test]
+    fn test_properties_64() {
+        let sixty_four_properties =
+            (0..64).map(|i: u8| (i.to_string(), serde_json::Value::Object(Map::new())));
+
+        let schema = serde_json::Value::Object(Map::from_iter([
+            (
+                "type".to_string(),
+                serde_json::Value::String("object".to_string()),
+            ),
+            (
+                "properties".to_string(),
+                serde_json::Value::Object(Map::from_iter(sixty_four_properties)),
+            ),
+        ]));
+
+        let curi = url::Url::parse("http://example/schema").unwrap();
+        let result = build_schema::<CoreAnnotation>(curi, &schema);
+
+        assert!(result.is_ok());
+        let kw = result.unwrap().kw;
+        assert!(matches!(kw[0], Keyword::Application(Properties { .. }, _)))
+    }
+
+    #[test]
+    fn test_properties_more_than_64() {
+        let sixty_four_properties =
+            (0..65).map(|i: u8| (i.to_string(), serde_json::Value::Object(Map::new())));
+
+        let schema = serde_json::Value::Object(Map::from_iter([
```

Comment:
Also just a nit: this would probably read nicer as

```rust
serde_json::json!({
  "type": "object",
  "properties": Map::from_iter(sixty_four_properties),
})
```